internal/config.go

```diff
@@ -143,3 +161,41 @@ func DefautConfigPath() string {
 	}
 	return filepath.Join(homeDirectoryPath, ".local/share/hz-cli", defaultConfigFilename)
 }
+
+func UpdateConfigWithSSL(config *hazelcast.Config, sslc *SSLConfig) error {
+	if sslc.ServerName != "" && sslc.InsecureSkipVerify {
+		return fmt.Errorf("SSL.ServerName and SSL.InsecureSkipVerify are mutually exclusive")
```

Comment:
I believe we should document that these are mut exclusive.

Reply:
You are right, we will add better documentation during refactoring.